src/hearings/containers/request-hearing/hearing-cancelled-summary/hearing-cancelled-summary.component.ts

```diff
@@ -0,0 +1,30 @@
+import { Component, OnDestroy } from '@angular/core';
+import { Store } from '@ngrx/store';
+import { HEARING_CANCELLED_SUMMARY_TEMPLATE } from '../../../../hearings/templates/hearing-cancelled-summary.template';
+import { ACTION, Mode } from '../../../models/hearings.enum';
+import { HearingsService } from '../../../services/hearings.service';
+import * as fromHearingStore from '../../../store';
+import { RequestHearingPageFlow } from '../request-hearing.page.flow';
+
+@Component({
+  selector: 'exui-hearing-cancelled-summary',
+  templateUrl: './hearing-cancelled-summary.component.html',
+})
+export class HearingCancelledSummaryComponent extends RequestHearingPageFlow implements OnDestroy {
```

Comment:
Is that better if we don't use this RequestHearingPageFlow? Use the back route directly on the parent summary page. Same as another component HearingCancellationSummaryComponent.